src/provider/hyperkzg.rs

```diff
@@ -61,18 +61,20 @@ where
   E::Fr: TranscriptReprTrait<E::G1>,
   E::G1Affine: TranscriptReprTrait<E::G1>, // TODO: this bound on DlogGroup is really unusable!
 {
-  fn compute_challenge(
+  /// TODO: write doc
+  pub fn compute_challenge(
     com: &[E::G1Affine],
     transcript: &mut impl TranscriptEngineTrait<NE>,
   ) -> E::Fr {
     transcript.absorb(b"c", &com.to_vec().as_slice());
     transcript.squeeze(b"c").unwrap()
   }
 
+  /// TODO: write doc
```

Comment:
Yes, so I do not focus on this only because we are going to merge Shplonk with HyperKZG eventually. The followup PR or "final" commit that you mentioned will for sure include these TODO resolvings